policy_set.go

```diff
@@ -136,6 +146,9 @@ type PolicySetCreateOptions struct {
 	// Optional: Whether or not the policy set is global.
 	Global *bool `jsonapi:"attr,global,omitempty"`
 
+	// Optional: The underlying technology that the policy set supports
+	Kind PolicyKind `jsonapi:"attr,kind,omitempty"`
```

Comment:
As this is optional shouldn't it be a `*PolicyKind` ?
```suggestion
	Kind *PolicyKind `jsonapi:"attr,kind,omitempty"`
```

Reply:
Note: From discussion with Glenn yesterday, moving to separate beta tests file. For customers that dont have MANAGED_OPA flag enabled this property will just be an empty string